os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_hp/ameba_ddrphy.c

```diff
@@ -200,7 +201,8 @@ void DDR_PHY_DLL_CLK_DIV(u32 DDR_Freq)
 	ddr_phy->DDRPHY_SSC3 = ((ddr_phy->DDRPHY_SSC3 & (~DDRPHY_MASK_DPI_N_CODE_T)) | DDRPHY_DPI_N_CODE_T(DDRPHY_Div_n_t));
 
 #if DDRPHY_SSC_EN
-	DDR_Freq = DDR_Freq * 99 / 100;
+	/* SSC=99% will let WIFI CH64(5310~5330) no spur, but degree 1.xDB */
```

Comment:
This kind of change is for new package?? If not, let's split commit.

Reply:
These are for DDR-related changes.